README.md

```diff
@@ -75,6 +75,13 @@ $details->hostname; // cpe-104-175-221-247.socal.res.rr.com
 $details->country; // US
 $details->country_name; // United States
 ```
+#### EU Country
+
+``Details->is_eU`` will return true if the country is a member of EU, as supplied by the ``eu.json`` file.
```

Comment:
`eU` or `eu`? This should really be `eu`, like imagine `is_http` vs. `is_hTTP` or something :smile:

Reply:
oh should be `eu` that's a typo there